src/pages/PageNotFound.tsx

```diff
@@ -0,0 +1,3 @@
+export const PageNotFound: React.FC = () => (
+  <h1 className="title">Page not found</h1>
+);
```

Comment:
Here we can use container div, to wrap our page.
It is not correct when you render the page and return only the title

```suggestion
export const PageNotFound: React.FC = () => (
  <div className="container">
    <h1 className="title">Page not found</h1>
   </div>
);
```